src/Service/SF1601/SF1601.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * This file is part of itk-dev/serviceplatformen.
+ *
+ * (c) 2020 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace ItkDev\Serviceplatformen\Service\SF1601;
+
+use DigitalPost\MeMo\Message;
+use DOMDocument;
+use DOMText;
+use ItkDev\Serviceplatformen\Service\AbstractRESTService;
+use ItkDev\Serviceplatformen\Service\Exception\ServiceException;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class SF1601 extends AbstractRESTService
+{
+    public const TYPE_DIGITAL_POST = 'Digital Post';
+
+    protected function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+        $resolver->setDefaults([
+            'svc_entity_id' => 'http://entityid.kombit.dk/service/kombipostafsend/1',
+            'svc_endpoint' => static function (Options $options) {
+                return $options['test_mode']
+                    ? 'https://exttest.serviceplatformen.dk/service/KombiPostAfsend_1/kombi'
+                    : 'https://prod.serviceplatformen.dk/services/KombiPostAfsend_1/kombi';
+            },
+        ]);
+    }
+
+    public function kombiPostAfsend(string $transactionId, string $type, Message $message)
+    {
+        $document = $this->buildKombiRequest($type, $message);
+
+//        header('content-type: text/plain'); echo var_export($document->saveXML(), true); die(__FILE__.':'.__LINE__.':'.__METHOD__);
+
+        $response = $this->call('POST', $this->getOption('svc_endpoint'), [
+                'headers' => [
+                    'content-type' => 'application/xml',
+                    'accept' => 'application/xml',
+                ],
+                'body' => $document->saveXML(),
+                'transactionId' => $transactionId,
+            ]);
+
+        return $response;
+    }
+
+    private function buildKombiRequest(string $type, Message $message)
+    {
+        if (!in_array($type, [self::TYPE_DIGITAL_POST])) {
```

Comment:
Is this made as preparation for other types? If not it seems unnecessary to use arrays for this.